### PR TITLE
EPMFARMATS-12598 ver. 2

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: EPMFARMATS-12598.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+const salaries = {
+    John: 100,
+    Ann: 160,
+    Pete: 130
+}
+
+function salaryCalculator(object) {
+    let sum = 0;
+    for (key in object) {
```
Comment (SiarheiRylach) [inline EPMFARMATS-12598.js:40]: let key in object

File: EPMFARMATS-12598.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+const menu = {
+    width: 200,
+    height: 300,
+    title: "My menu"
+};
+
+function multiplyNumeric(object) {
+    for (key in object) {
+        if (typeof (object[key]) == "number") object[key] *=  2;
```
Comment (SiarheiRylach) [inline EPMFARMATS-12598.js:57]: Why do you use parentheses here?

Comment (overdd) [reply EPMFARMATS-12598.js:57]: It's double check due to fear of unexpected behavior. typeof(x) and typeof x are both correct according to https://javascript.ru/typeof

**Comment by overdd** [comment]
Issues are fixed

**Pull request merged**